backends/portfolio_doc/tasks.rb

```diff
@@ -0,0 +1,122 @@
+# frozen_string_literal: true
```

Comment:
What's this Rake file for? I think most of it is already present in `backends/crd_doc/tasks.rake`.

Reply:
It was a work in progress. I've actually deleted it in my latest workspace. So, can we just ignore this for now and approve this PR since I'll be submitting a new PR for #237 soon anyways (which removes this file).